### PR TITLE
Update the syntax of CSS `sin()` function

### PR DESCRIPTION
**Pull request by ramiy**
#### Summary
Add syntax examples including`<angle>` values, `<number>` values and other values (`pi` and `e`).

#### Motivation
The change help readers to see all the available arguments the function receives.

#### Metadata
- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/sin/index.md [hunk 1]
```diff
@@ -29,6 +39,10 @@ The `sin()` function takes only one expression as its argument.
 
 {{CSSSyntax}}
 
+```css
+sin( [ {{cssxref("&lt;angle&gt;")}} || {{cssxref("&lt;number&gt;")}} ] )
+```
+
```
[the remark this answers is not in the document]
Comment (ramiy) [reply files/en-us/web/css/sin/index.md:45]: Added to CSS syntax: https://github.com/mdn/data/pull/592

File: files/en-us/web/css/sin/index.md [hunk 2]
```diff
@@ -15,12 +14,23 @@ spec-urls: https://drafts.csswg.org/css-values/#trig-funcs
...
 ```css
-width: calc( sin(-45deg) * 1px );
+/* Single <angle> values */
+width: calc( sin(45deg) * 100px );
+width: calc( sin(0.25turn) * 100px );
+width: calc( sin(1.0471967rad) * 100px );
+
+/* Single <number> values */
+width: calc( sin(3.141592653589793) * 100px ); /* pi */
+width: calc( sin(1.57079632679) * 100px ); /* pi / 2 */
```
Comment (teoli2003) [inline files/en-us/web/css/sin/index.md:29]: I'm not sure we should imply that converting `pi` into its numerical value is a good idea. Let's use a random value.

π is better represented as `pi` or `0.5turn`.

```suggestion
width: calc( sin(63.673) * 100px );
width: calc( sin(2 * 0.125) * 100px );
```

File: files/en-us/web/css/sin/index.md [hunk 3]
```diff
@@ -15,12 +14,23 @@ spec-urls: https://drafts.csswg.org/css-values/#trig-funcs
...
+
+/* Other values */
+width: calc( sin(pi/2) * 100px );
+width: calc( sin(e/4) * 100px );
```
Comment (teoli2003) [inline files/en-us/web/css/sin/index.md:33]:
```suggestion
width: calc( sin(pi / 2) * 100px );
width: calc( sin(e / 4) * 100px );
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/web/css/sin/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ramiy) [reply files/en-us/web/css/sin/index.md]: Also updated csstree: https://github.com/csstree/csstree/pull/200

[the remark this answers is not in the document]
Comment (wbamberg) [reply files/en-us/web/css/sin/index.md]: This also seems to be part of https://github.com/mdn/content/issues/18780#issuecomment-1208788278 ?

**Pull request merged**